src/dftbp/dftbplus/initprogram.F90

```diff
@@ -5917,6 +6021,10 @@ subroutine ensureHybridXcReqs(this, tShellResolved, hybridXcInp)
       call error("Global hybrid functionals not currently implemented for linear response.")
     end if
 
+    if (this%tQuadrupole) then
```

Comment:
Also external electric fields/charges should be blocked somewhere as the field / field gradients will couple with dipole / quadrupole terms.